docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -63,7 +63,7 @@ Returns the short string `'VALID'` if valid, otherwise it reverts.
 
 [.contract]
 [[AccountComponent]]
-=== `++AccountComponent++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.8.0/src/account/account.cairo#L27[{github-icon},role=heading-link]
+=== `++AccountComponent++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-vX.Y.Z/src/account/account/account.cairo[{github-icon},role=heading-link]
```

Comment:
Both that and avoiding someone getting to that file some way to think that's the version matching the actual release.

Reply:
> Both that and ...

it's the other way around, using the current version streamlines the version bump since it's a single replace across the entire codebase. I'm not convinced the latter should weigh too much

Reply:
True. I don't have a strong case to defend it, I will update them back then. cc @andrew-fleming